### DISCUSSION TITLE
Clojure Support

### DISCUSSION THREAD

**Discussion by prcastro** [zed-industries/zed#5814] [Feature Requests]
I'm thinking of implementing Clojure support on Zed. Can I give it a shot? I suppose it's something of interest for the editor, is that right?

**Comment by prcastro** [comment]
https://github.com/zed-industries/zed/pull/6988

I'm just having problems of communication between the server and the editor

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
